powdr/src/lib.rs

```diff
@@ -112,12 +112,14 @@ impl Session {
         let vkey = Path::new(&self.out_path).join(DEFAULT_VKEY);
 
         if generate_artifacts {
+            println!("Creating program ZK setup. This has to be done only once per program.");
```

Comment:
> "This has to be done only once per program."
This is the kind of stuff that should printed when you run `powdr --help".

But it is good to have it here, too.